### PR TITLE
Added icons for Create and Login option

### LINKED ISSUE #3466: Replacing text with their icons on navigation bar
**Actual Behaviour**
Replacing the text `Create ` and `Login` on navigation bar.

**Expected Behaviour**
Added their icons

**Steps to reproduce it**
1. See Error at the navigation bar

**Screenshots of the issue**

Screenshot 2020-02-14 01:32:58

Screenshot 2020-02-14 01:33:36

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mohitm15**
Fixes #3466 [Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]

Changes: [Add here what changes were made in this pull request and if possible provide links showcasing the changes.]
Added the icons for option `create ` and  `login`.

Screenshots of the change: [Add screenshots depicting the changes.]

**Before**
Screenshot 2020-02-14 01:32:58

Screenshot 2020-02-14 01:33:36

**After**
Screenshot 2020-02-14 01:02:26
Screenshot 2020-02-14 01:01:43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "@material-ui/lab": "3.0.0-alpha.30",
     "add-px": "1.0.0",
     "axios": "0.19.0",
+    "babel-eslint": "^10.0.3",
```
Comment (snitin315) [inline package.json]: Why???? This change is not related to issue.

File: src/components/NavigationBar/index.js [hunk 2]
```diff
@@ -674,7 +675,14 @@ class NavigationBar extends Component {
                             }}
                           />
                         ) : (
-                          <CreateDetail>Create</CreateDetail>
+                          <CreateDetail>
+                            <IconButton
+                              color="inherit"
+                              style={{ padding: '7px' }}
```
Comment (snitin315) [inline src/components/NavigationBar/index.js]: Avoid Inline styling.

File: package.json (hunk 1 above)
Comment (mohitm15) [reply package.json]: Added `babel es-linter` as earlier I was unable to commit due to missing of same package. That's why added.

File: src/components/NavigationBar/index.js (hunk 2 above)
Comment (mohitm15) [reply src/components/NavigationBar/index.js]: Okay I will remove that.

**Comment by tobehonest** [comment]
@mohitm15 Please close this PR as issue is closed now.

**Comment by Orbiter** [comment]
closing this as requested by @mohitm15
